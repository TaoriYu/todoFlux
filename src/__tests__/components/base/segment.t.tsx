import * as React from 'react';
import renderer = require('react-test-renderer');
import {Segment} from 'semantic-ui-react';

describe('Segment component', () => {
  test('basic segment snapshot', () => {
    expect(renderer.create(<Segment>I'm full well aware</Segment>).toJSON()).toMatchSnapshot();
  });
});