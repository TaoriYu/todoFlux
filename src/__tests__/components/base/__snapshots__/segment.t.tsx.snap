// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Segment component basic segment snapshot 1`] = `
<div
  className="ui segment"
>
  I'm full well aware
</div>
`;
